base_continent/data/continent_data.xml

```diff
@@ -1,27 +1,37 @@
 <?xml version="1.0" encoding="utf-8"?>
-<openerp>
+<odoo>
+
     <data noupdate="1">
```

Comment:
```<data>``` is deprecated. Put the noupdate directly on the odoo element. Remove data element and move the indentation of the rest.